pkg/kubectl/explain/explain_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package explain
+
+import (
+	"reflect"
+	"testing"
+
+	cmdtesting "k8s.io/kubernetes/pkg/kubectl/cmd/testing"
+)
+
+func TestSplitAndParseResourceRequest(t *testing.T) {
+	f, _, _, _ := cmdtesting.NewTestFactory()
+	mapper, _ := f.Object()
```

Comment:
minor nit - would be nice to contain scope of "f" with a small function, i.e.
mapper  := getMapper()
where getMapper hid f, the call to f.object, etc

Reply:
Done.